tuned/daemon/daemon.py

```diff
@@ -122,11 +122,18 @@ def _notify_profile_changed(self, profile_names, result, errstr):
 		return errstr
 
 	def _full_rollback_required(self):
-		retcode, out = self._cmd.execute(["systemctl", "is-system-running"], no_errors = [0])
+		retcode, out, err_msg = self._cmd.execute(["systemctl", "is-system-running"], no_errors = [0], return_err = True)
 		if retcode < 0:
 			return False
+
+		# if there is some stderr and not stdout, it means systemctl runs in container
```

Comment:
I believe that for "containers" this is a wrong assumption to make and I have no idea how I'd test for "containers" since "containers" is such a wide term.  Anyways, this "works" in openshift-tuned container shipped in OCP 4.2:

Running "in a container" 1)
```
sh-4.2# systemctl is-system-running >/tmp/stdout 2>/tmp/stderr ; echo $?
0
sh-4.2# cat /tmp/stdout
sh-4.2# cat /tmp/stderr
Running in chroot, ignoring request.
```
but this does not:

Running "in a container" 2)
```
sh-4.2# SYSTEMD_IGNORE_CHROOT=1 systemctl is-system-running >/tmp/stdout 2>/tmp/stderr ; echo $?
0
sh-4.2# cat /tmp/stdout
running
sh-4.2# cat /tmp/stderr
sh-4.2# 
```

You can probably detect you're running in a chroot()ed env though if you make sure `SYSTEMD_IGNORE_CHROOT=0`.  The question is, whether this is what we want.  I really only do not want to regress at this point.  If we keep this code, the comment should be rephrased and very likely a search for "Running in chroot" introduced not to make any wrong assumptions.  Will that error message "Running in chroot" never change though?  This all seems very brittle.  Can someone remind me exactly where is the original code broken given one uses `SYSTEMD_IGNORE_CHROOT=1` when using tuned?  BTW, I do not use `SYSTEMD_IGNORE_CHROOT=1` when running tuned in a container, so perhaps I should fix my stuff?  Thanks!